### ISSUE TITLE
Disable super_clipboard for a platform?

### ISSUE THREAD

**Issue by mreinfurt** [superlistapp/super_native_extensions#78]
Hello and first of all, thanks for the great package! It's been working really well and was very easy to integrate.

In our project, we target iOS, Android and Web. We wanted to implement a Copy & Paste feature mainly on Web. Due to NDK being needed for Android, we wanted to disable the feature on Android so that we don't need to rely on the NDK. However, as soon as we have super_clipboard in pubspec, Android will not build due to NDK issues.

Therefore I wanted to ask: Is it somehow possible to disable super_clipboard on a platform? (I would like to avoid flavors to reduce complexity).

Thank you!

**Comment by knopp** [comment]
I don't think this is currently possible with pub. There is no way to configure per platform dependencies. NDK installation is should be quite straightforward though.

In any case, this seems like a question better suited for Dart/Flutter.

**Comment by knopp** [comment]
Is this on a CI or local?

**Comment by knopp** [comment]
Also, does `ulimit -n 65536` not help at all?

**Comment by knopp** [comment]
It is a bit weird though because I have file descriptors limited to 256 (default) and have never seen the error.

**Comment by mreinfurt** [comment]
It's local. And yeah, for some reason `ulimit -n 65535` doesn't help. I also tried other fixes related to file descriptors (because it seems on macOS 65536 might not be accepted and limit must be lower and so on) but no fix worked. I found some related issues like [this one](https://github.com/servo/servo/issues/23409) but nothing worked so far. 

The build works on other machines (although none of them are running a Mac with M1/M2 chip). I'll probably investigate a bit more next week.
